src/styles/styles.js

```diff
@@ -2878,6 +2878,48 @@ const styles = {
         paddingTop: 80,
         paddingBottom: 45,
     },
+
+    footerWrapper: {
+        backgroundColor: themeColors.midtone,
+        fontSize: variables.fontSizeNormal,
+        paddingTop: 64,
+        paddingLeft: 17,
```

Comment:
What does the left padding of 17 do exactly?

Reply:
I was having a hard time getting the entire container to center properly on both mobile and web. When I would use align-items/justify-content the content would get squished. If I keep this method, this should probably be 16px since that's a nicer number... but here's a video to show what it's doing

https://user-images.githubusercontent.com/38015950/218583156-9b5edb26-858b-425a-8085-8a4e5dba8288.mov

Reply:
It looks like it works well without the padding left though?

<img width="1141" alt="image" src="https://user-images.githubusercontent.com/2319350/218608038-e0e84845-2ebe-4ab7-81c0-eb737a5609cd.png">